src/Controller/AbstractController.php

```diff
@@ -61,7 +63,7 @@ private function renderContent(string $view, array $parameters = []): string
 
     public function getViewPath(): string
     {
-        return $this->aliases->get('@views') . '/' . $this->name();
+        return $this->aliases->get('@views') . '/' . self::getName();
```

Comment:
Better to use static in order to allow changing default algorithm.

Reply:
Yes sure